guides/installing-python.md

```diff
@@ -0,0 +1,147 @@
+# Installing and using Python 2 and 3
+
+- [Migrating from Homebrew-installed Python](#migrating-from-homebrew-installed-python)
+   - [Remove Homebrew-installed virtualenvwrapper](#remove-homebrew-installed-virtualenvwrapper)
+   - [Optional: remove Homebrew-installed Python and variables](#optional-remove-homebrew-installed-python-and-variables)
+- [Installing pyenv](#installing-pyenv)
+- [Configuring pyenv](#configuring-pyenv)
+- [Installing Python](#installing-python)
+- [Setting the global Python versions](#setting-the-global-python-versions)
+- [Using local Python versions](#using-local-python-versions)
+- [Upgrading Python versions](#upgrading-python-versions)
+- [Further reading](#further-reading)
+
+## Migrating from Homebrew-installed Python
+
+**Note**: Any virtualenvs that you have prior to this process will have to be recreated to use Python from pyenv.
+
+### Remove Homebrew-installed virtualenvwrapper
+
+Before installing and configuring pyenv, uninstall virtualenvwrapper if it  is installed via Homebrew:
+
+```shell
+brew uninstall virtualenvwrapper
+```
+
+Then remove the sourcing of `virtualenvwrapper.sh` and related variables from your shell's dotfiles (`.profile`, `.bashrc` for bash users, `.zshrc` for zsh users):
+
+```shell
+export VIRTUALENVWRAPPER_PYTHON=$HOME/homebrew/bin/python
+export VIRTUALENVWRAPPER_VIRTUALENV=$HOME/homebrew/bin/virtualenv
+source $HOME/homebrew/bin/virtualenvwrapper.sh
+```
+
+### Optional: remove Homebrew-installed Python and variables
+
+Before installing and configuring pyenv, it is a good idea to uninstall all old versions of Python and virtualenvwrapper that might be installed via Homebrew, and remove any varaibles from your shell's dotfiles (`.profile`, `.bashrc` for bash users, `.zshrc` for zsh users):
+
+```shell
+brew uninstall python
+brew cleanup python
+brew uninstall virtualenvwrapper
+```
+You should also remove any variables that reference Homebrew's Python from your shell's dotfiles (`.profile`, `.bashrc` for bash users, `.zshrc` for zsh users), for example:
```

Comment:
To emphasize the "remove" instruction, I'd change it to:

"for example, removing:"

Reply:
Added in 33495a5.